db/models.py

```diff
@@ -50,3 +59,65 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=User, on_delete=models.CASCADE, related_name="orders"
+    )
+
+    def __str__(self) -> str:
+        return f"{self.created_at.strftime("%Y-%m-%d %H:%M:%S")}"
```

Comment:
Just a tip: there's no need to use a f-string in here, since the strftime method already returns a string.

Reply:
Also, I just noticed that there's a flake8 complaint in the pipeline of your code. To solve it, just remove the f-string as I mentioned earlier, or do this:

```suggestion
        return f"{self.created_at.strftime('%Y-%m-%d %H:%M:%S')}"
```